app/interactors/export_mappings.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+class ExportMappings
+  include Interactor
+
+  delegate :available_mappings, :filter, :format, :mapping_id, to: :context
```

Comment:
Would be great to add some comments, especially on mapping_id as it's not clear what is it for, why we're excluding it from mappings and construct filename not based on it.

Reply:
I refactored the interactor, hopefully it makes more sense now.